### ISSUE TITLE
Add filter option to each page in docs

### ISSUE THREAD

**Issue by givensuman** [react-icons/react-icons#655]
**Is your feature request related to a problem? Please describe.**
There is a global filter for searching for icons, but no implemented method of searching for icons within a selected icon library.

**Describe the solution you'd like**
Add an input to the top of each icon library's page and filter the icons loaded on the page based on input.

**Describe alternatives you've considered**
Currently `Ctrl` + `F` is the workaround.

**Comment by zachgoll** [comment]
Came here to add this same feature request!  This would be super helpful!

**Comment by CrowBe** [comment]
Agreed, the existing search input is still on the page so it would seem straightforward to implement. The current functionality is that typing within the search input overwrites filtering by library which is a bit confusing. Applying both would be really nice. Would be happy to do a PR but the docs indicated PR's for that part of the repo are not accepted.
